src/program.js

```diff
@@ -601,6 +601,11 @@ Example: $0 --help run.
             'Only used with `use-submission-api`',
           type: 'string',
         },
+        'version-source': {
+          describe:
+            'Path to a zip file containing human readable source code for a version. ' +
+            'Only used with `use-submission-api`',
```

Comment:
type:string is missing here.

(also demandOption, like the other params)

Reply:
`demandOption` is inconsistent here and some of the other options don't specify either - it defaults to `false` so maybe we should just omit for non-required cases?

Reply:
Let's not worry about demandOption for now then.